src/bbs_create_proof.rs

```diff
@@ -17,6 +17,32 @@ struct CreateProofContext {
     nonce: Option<ProofNonce>,
 }
 
+struct USize(usize);
```

Comment:
Could we please put a description here documenting what we think is the problem that creates the need for this work around @mikelodder7?